pkg/process/logging.go

```diff
@@ -40,6 +41,15 @@ func newLogger() (*zap.Logger, error) {
 		levelEncoder = zapcore.CapitalLevelEncoder
 	}
 
+	winFileSink := func(u *url.URL) (zap.Sink, error) {
+		// Remove leading slash left by url.Parse()
+		return os.OpenFile(u.Path[1:], os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0644)
+	}
+	err := zap.RegisterSink("winfile", winFileSink)
```

Comment:
should this happen in a `func init()` rather than `newLogger`?

Reply:
yes, it needs to be registered only once so `init` will be better.

Reply:
hmm should be throw `fatal` in case of error in `init()`?

Reply:
`panic` is pretty much the only option unfortunately